frontend/src/app/testResults/uploads/CsvUploadGuide.tsx

```diff
@@ -0,0 +1,271 @@
+                  </a>
+                </li>
+              </ul>
+            </section>
+            <section>
+              <h3
+                id="preparing-a-csv"
+                className="font-body-lg border-top-1px border-ink margin-top-8 margin-bottom-6 padding-top-1"
+              >
+                Preparing a CSV file
+              </h3>
+              <ol className="usa-process-list">
+                <li className="usa-process-list__item">
+                  <h4 className="usa-process-list__heading">
+                    Download the SimpleReport reference CSV and review the
+                    documentation
+                  </h4>
+                  <p className="margin-top-05">
+                    If your jurisdiction already has a set format for CSV,
+                    compare it to our{" "}
+                    <a
+                      href="https://SimpleReport.cdc.gov/assets/csv/SimpleReport-StandardCSV-ExampleData-20220509.csv"
+                      className="usa-link"
+                    >
+                      standard CSV file
+                    </a>{" "}
+                    and{" "}
+                    <a href="/getting-started/testing-facilities/csv-schema">
+                      {" "}
+                      CSV schema documentation
+                    </a>
+                    .
+                  </p>
+                  <p>
+                    If you're starting from scratch, use the standard CSV as a
+                    template.
+                  </p>
+                </li>
+                <li className="usa-process-list__item">
+                  <h4 className="usa-process-list__heading">
+                    Format your column headers to match the SimpleReport
+                    standard CSV
+                  </h4>
+                  <p>
+                    Whether you're modifying an existing file or creating a new
+                    one from scratch, be sure to include all column headers
+                    outlined in the standard file and documentation. Do not
+                    include any additional column headers. Make sure headers are
+                    written exactly as defined.
+                  </p>
+                  <p>
+                    Include column headers even if you don’t have data for every
+                    field.
+                  </p>
+                </li>
+                <li className="usa-process-list__item">
+                  <h4 className="usa-process-list__heading">
+                    Enter values into your CSV file
+                  </h4>
+                  <p>
+                    Using the{" "}
+                    <a href="/getting-started/testing-facilities/csv-schema">
+                      {" "}
+                      CSV schema documentation
+                    </a>{" "}
+                    as a guide, enter properly formatted values in the relevant
+                    fields.
+                  </p>
```

Comment:
```suggestion
                  <p>
                    Following the{" "}
                    <a href="/getting-started/testing-facilities/csv-schema">
                      {" "}
                      CSV template documentation
                    </a>{" "}
                   , enter properly formatted values in the relevant
                    fields.
                  </p>
```
Recommend renaming "CSV schema documentation" to "CSV template guide" across the board, since some people we tested with were unfamiliar with the term "schema."